docs/sources/alerting/configure-notifications/manage-contact-points/integrations/configure-email.md

```diff
@@ -0,0 +1,94 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/alerting/configure-notifications/manage-contact-points/integrations/configure-email/
+description: Configure the Email integration to connect alerts generated by Grafana Alerting
+keywords:
+  - grafana
+  - alerting
+  - email
+  - integration
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+menuTitle: Email
+title: Configure Email for Alerting
+weight: 300
+---
+
+## Configure Email for Alerting
+
+Use the Grafana Alerting - email integration to send email notifications when your alerts are firing. An email is sent when an alert fires and when an alert gets resolved.
+
+## Before you begin
+
+When utilizing Grafana OSS, enabling email notifications requires configuring [SMTP settings][smtp-settings] in the Grafana configuration settings. In contrast, for users of Grafana Cloud, SMTP configuration is not required.
+
+### SMTP configuration
+
+1. **Access the Configuration File.**
+
+   Locate the Grafana configuration file. This file is typically named `grafana.ini` or `custom.ini` and is located in the `conf` directory within the Grafana installation directory.
+
+1. **Open the Configuration File:**
+
+   Open the configuration file using a text editor.
+
+1. **Locate SMTP Settings Section.**
+
+   Search for the [SMTP settings section][smtp-settings] in the configuration file. It starts with `[smtp]`.
+
+1. **Configure SMTP Settings.**
+
+   Within the `[smtp]` settings section, specify the following parameters:
+
+   - `enabled = true`: Enables SMTP.
+   - `host`: The hostname or IP address of your SMTP server, and the port number of your SMTP server (commonly 25, 465, or 587). Default is `localhost:25`.
+   - `user`: Your SMTP username (if authentication is required).
+   - `password`: Your SMTP password (if authentication is required).
+   - `from_address`: The email address from which Grafana notifications will be sent.
+   - `from_name`: The name associated with the from_address.
+   - `skip_verify = true`: Skip SSL/TLS certificate verification (useful for testing, but not recommended for production).
+
+1. **Save and Close the Configuration File.**
```

Comment:
same comment as above

Reply:
ok here at least i will keep the code format so readers differentiate each element

Reply:
never mind, i see now you meant the  
1. **Save and Close the Configuration File.**


 not the above text